.eslintrc

```diff
@@ -55,7 +54,7 @@
     "no-negated-in-lhs": 2,
     "no-obj-calls": 2,
     "no-regex-spaces": 2,
-    "quote-props": [2, "as-needed", {"keywords": true}],
+    "quote-props": [2, "as-needed", {"keywords": true, "unnecessary": false}],
```

Comment:
Ok, I think this is the last question I have so far. { "unnecessary": false } seems to be a relaxation of the current rules. Was it the default in the previous version? Did you have compile errors, without this option?